src/Caching/Storages/SQLiteJournal.php

```diff
@@ -32,7 +32,8 @@ public function __construct($path)
 		$this->pdo = new \PDO('sqlite:' . $path);
 		$this->pdo->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
 		$this->pdo->exec('
-			PRAGMA foreign_keys = ON;
+			PRAGMA foreign_keys = OFF;
+			PRAGMA journal_mode = WAL;
```

Comment:
Good idea. SQLite 3.7 should be there since PHP 5.4. Personally, I would add `PRAGMA synchronous = OFF;` too.